src/parser/expression.js

```diff
@@ -385,6 +385,16 @@ pp.parseExprAtom = function (refShorthandDefaultPos) {
       }
       return this.finishNode(node, "Super");
 
+    case tt._import:
+      if (!this.hasPlugin("importFunctions")) this.unexpected();
+
+      node = this.startNode();
+      this.next();
+      if (!this.match(tt.parenL)) {
```

Comment:
I figured out that this can be hit if you do something that parses as an expression first. For example, inside of a function `return import.fails()` hits this case.

I've added a test for this case to prevent any regressions and to bump the code coverage.